src/Fantomas.Core/SyntaxOak.fs

```diff
@@ -766,49 +766,64 @@ type RecordFieldNode(fieldName: IdentListNode, equals: SingleTextNode, expr: Exp
     member val Equals = equals
     member val Expr = expr
 
-type ExprRecordNode
+[<AbstractClass>]
+type ExprRecordNode(openingBrace: SingleTextNode, fields: RecordFieldNode list, closingBrace: SingleTextNode, range) =
+    inherit NodeBase(range)
+
+    member val OpeningBrace = openingBrace
+    member val Fields = fields
+    member val ClosingBrace = closingBrace
+    member x.HasFields = List.isNotEmpty x.Fields
+
+type ExprCopyableRecordNode
     (
         openingBrace: SingleTextNode,
-        extra: RecordNodeExtra,
```

Comment:
`RecordNodeExtra` is no longer being used, so please remove that type all together.